atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraLogHelper.java

```diff
@@ -72,6 +72,14 @@ private static String getUpperEndpoint(Range<LightweightOppToken> range) {
         return range.upperEndpoint().toString();
     }
 
+    public static List<String> tokenRangeHashes(Set<TokenRange> tokenRanges) {
+        return tokenRanges.stream()
+                .map(range -> String.format(
+                        "(%d, %d)",
+                        range.getStart_token().hashCode(), range.getEnd_token().hashCode()))
```

Comment:
might want to avoid the String.format to avoid CPU & GC thrash
```suggestion
                .map(range -> "(" + range.getStart_token().hashCode() + ',' + range.getEnd_token().hashCode() + ")")
```

Reply:
makes sense, thanks!